### PR TITLE
[Wallet][RPC] add missing abandon status documentation

### PR DESCRIPTION
**Pull request by jonasschnelli**

### CODE REVIEW

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -1439,6 +1439,7 @@ UniValue listtransactions(const UniValue& params, bool fHelp)
             "    \"vout\": n,                (numeric) the vout value\n"
             "    \"fee\": x.xxx,             (numeric) The amount of the fee in " + CURRENCY_UNIT + ". This is negative and only available for the \n"
             "                                         'send' category of transactions.\n"
+            "    \"abandoned\": xxx          (bool) Wether the transaction has been abandoned (inputs are respenable).
```
Comment (paveljanik) [inline src/wallet/rpcwallet.cpp]: whether (shouldn't it be "If" instead?), respendable.

**Comment by laanwj** [comment]
```
wallet/rpcwallet.cpp:1442:13: warning: missing terminating " character [enabled by default]
wallet/rpcwallet.cpp:1442:13: error: missing terminating " character
```

**Comment by jonasschnelli** [comment]
> `wallet/rpcwallet.cpp:1442:13: warning: missing terminating " character [enabled by default]`
> `wallet/rpcwallet.cpp:1442:13: error: missing terminating " character`

Fixed.
This is what you get when you think you don't need to compile simple text changes!

File: src/wallet/rpcwallet.cpp [hunk 2]
```diff
@@ -1439,6 +1439,7 @@ UniValue listtransactions(const UniValue& params, bool fHelp)
             "    \"vout\": n,                (numeric) the vout value\n"
             "    \"fee\": x.xxx,             (numeric) The amount of the fee in " + CURRENCY_UNIT + ". This is negative and only available for the \n"
             "                                         'send' category of transactions.\n"
+            "    \"abandoned\": xxx          (bool) 'true' if the transaction has been abandoned (inputs are respenable).\n"
```
Comment (laanwj) [inline src/wallet/rpcwallet.cpp]: s/respenable/respendable/

**Comment by laanwj** [comment]
> This is what you get when you think you don't need to compile simple text changes!

Hehe, yes, my experience too! Rushed changed usually take at least three times as long :)

**Comment by jonasschnelli** [comment]
Fixed the typo.

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/7742/commits/cef8bdf5d747d42c3be473d7ef38052eebd5e9bd

**Pull request merged**